generate_eval_config.py

```diff
@@ -416,6 +420,14 @@ def main(
 
     new_config["tasks"] = tasks_dict
 
+    # Extract actual values from dict-wrapped configs
```

Comment:
It's a fix to an issue I've had randomly pop up with retrieval configs. It could be its own PR but I figured I'd bundle it with this since that's where I had the issue pop up the most.

Reply:
Totally fine, just wanted to make sure it was not stage by accident